deploy/runtools/runtime_config.py

```diff
@@ -72,30 +83,37 @@ def get_design_name(self) -> str:
 
     def get_local_driver_binaryname(self) -> str:
         """ Get the name of the driver binary. """
-        return self.get_design_name() + "-" + self.platform
+        return self.driver_name_prefix + self.get_design_name() + self.driver_name_suffix
+
+    def get_local_driver_dir(self) -> str:
+        """ Get the relative local directory that contains the driver used to
+        run this sim. """
+        return self.local_driver_base_dir + "/" + self.platform + "/" + self.get_deploytriplet_for_config() + "/"
 
     def get_local_driver_path(self) -> str:
         """ return relative local path of the driver used to run this sim. """
-        my_deploytriplet = self.get_deploytriplet_for_config()
-        drivers_software_base = LOCAL_DRIVERS_BASE + "/" + self.platform + "/" + my_deploytriplet + "/"
-        fpga_driver_local = drivers_software_base + self.get_local_driver_binaryname()
-        return fpga_driver_local
+        return self.get_local_driver_dir() + self.get_local_driver_binaryname()
 
     def get_local_runtimeconf_binaryname(self) -> str:
         """ Get the name of the runtimeconf file. """
-        return "runtime.conf" if self.customruntimeconfig is None else os.path.basename(self.customruntimeconfig)
+        return self.get_design_name() + "-generated.runtime.conf" if self.customruntimeconfig is None else os.path.basename(self.customruntimeconfig)
```

Comment:
The base filename is Hardcoded to `FireSim-generated` across the makefiles.

Reply:
(So i would just hard code it here).

Reply:
fixed